src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -623,19 +635,16 @@ bool fx_muxer_t::resolve_sdk_dotnet_path(const pal::string_t& own_dir, pal::stri
             }
         }
     }
-    else
-    {
-        trace::verbose(_X("Failed to obtain current working dir"));
-    }
 
     std::vector<pal::string_t> hive_dir;
     pal::string_t local_dir;
     pal::string_t global_dir;
-    bool multilevel_lookup = multilevel_lookup_enabled();
+
+    bool multilevel_lookup = own_dir.empty() ? true : multilevel_lookup_enabled();
```

Comment:
AFAICT, without multilevel lookup, this resolution with no own_dir would always fail. That seems useless.